source/connect.txt

```diff
@@ -9,5 +9,174 @@ Connect to a Deployment
+
+.. example:: 
+
+   To connect to a MongoDB
+   instance running on a remote host machine on port 28015:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com:28015"
+
+.. note:: Connecting to |service|
+
+   If your remote host is a |service-fullname| cluster, you can copy 
+   your connection string from the |service| UI. To learn more, see 
+   :atlas:`Connect to a Cluster <connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+
+Connection Considerations
+-------------------------
+
+Authentication
+~~~~~~~~~~~~~~
+
+To connect to a MongoDB instance requires authentication, use the 
+``--username`` and ``--authenticationDatabase`` command-line options. 
+``mongosh`` prompts you for a password, which it masks as you type.
+  
+  .. example:: 
+
+     To connect to a remote MongoDB instance and authenticate against 
+     the ``admin`` database as user ``alice``:
+
+     .. code-block:: sh
+
+        mongosh "mongodb://mongodb0.example.com:28015" --username alice --authenticationDatabase admin
+
+  .. note::
+
+     To provide a password with the command instead of using the masked 
+     prompt, you can use the ``--password`` option.
+
+.. seealso::
+
+   - :manual:`Enable Access Control </tutorial/enable-authentication/>` 
+     to enforce authentication.
+   - Add :manual:`Database Users </core/security-users/>` 
+     to provide authenticated access to a MongoDB instance.
+
+Connecting to a Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a replica set:
+
+- If you are using the :manual:`DNS Seedlist Connection Format 
+  </reference/connection-string/#dns-seedlist-connection-format>`, you 
+  can include the ``+srv`` modifier in your connection string.
+
+  .. example::
+
+     .. code-block:: sh
+
+        mongosh "mongodb+srv://server.example.com/"
+
+  .. note::
+     Using the ``+srv`` connection string modifier automatically sets 
+     the 
+     :manual:`tls option </reference/connection-string/#urioption.tls>` 
+     to ``true`` for the connection. You can 
+     override this behavior by explicitly setting ``tls`` to ``false``.
+
+- You can specify the replica set name and members in the
+  :manual:`connection string </reference/connection-string>`.
+
+  .. example::
+
+     To connect to a three-member replica set named ``replA``:
+
+     .. code-block:: sh
+
+        mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
+
+TLS Connections
+~~~~~~~~~~~~~~~
+
+For ``tls`` connections:
+
+- You can use the 
+  :manual:`tls option </reference/connection-string/#urioption.tls>` to 
+  set ``tls=true`` in the 
+  :manual:`connection string </reference/connection-string>`:
+
+  .. example::
+
+     To enable ``tls`` in the connection string:
+
+     .. code-block:: sh
+
+        mongosh "mongodb://mongodb0.example.com:28015/?tls=true"
+
+- If you are using the :manual:`DNS Seedlist Connection Format 
```

Comment:
Do we want to move this DNS example the top to match the other section where we made DNS first?

Reply:
Yes, good call